RO/Country_fiches/RO-test-Country_fiches-2024.md

```diff
@@ -0,0 +1,68 @@
+# _RO_ - _2024_: Country Fiche
+
+## Table of Contents
+1. [Introduction](#introduction)
+1. [State of Play](#state_of_play)
+   1. [Coordination](#Coordination)
+   2. [Functioning and coordination of the infrastructure](#functioning)
+   3. [Usage of the infrastructure for spatial information](#usage)
+   4. [Data Sharing Arrangements](#data)
+   5. [Costs and Benefits](#costs)
+
+
+## Introduction
+
+< insert text >
+
+## State of Play
+
+< insert text >
+
+### Coordination
+
+#### National Contact Point
+
+- Name of Public Authority: National Agency for Cadastre and Land Registration
+- Postal Address: No. 202A Splaiul Independenței, 1st floor, sector 6, Bucharest
+- Contact Email:
+- Telephone Number:
+- Telefax Number:
+- National INSPIRE Website:
+- MIG Contacts: 
+  - Contact Person:
+  - Email:
+  - Telephone Number:
+  - Contact Person:
+  - Email:
+- MIG T Contacts: 
+  - Contact Person:
+  - Email:
+  - Contact Person:
+  - Email:
+  - Telephone Number:
+
+#### Coordination Structure & Progress: 
+
+##### Coordination structure
+
+< insert text >
+
+##### Progress
+
+< insert text >
+
+### Functioning and coordination of the infrastructure <a name="functioning"></a>
+
+< insert text >
+
+### Usage of the infrastructure for spatial information <a name="usage"></a>
+
+< insert text >
+
+### Data Sharing Arrangements <a name="data"></a>
+
+< insert text >
+
+### Costs and Benefits <a name="costs"></a>
+
+< insert text >
```

Comment:
Key facts and figures

Key facts and figures from monitoring 2023 can be found at:
#https://inspire-geoportal.ec.europa.eu/catalog/views/inspireportal/monitoringreporting/site/mr2023_details.html?country=ro